R/test_getting_started.R

```diff
@@ -24,7 +24,8 @@ test_getting_started <- function(skip = NULL, ctx = get_default_context()) {
     #' \code{make_context()} must be called before calling any of the
     #' \code{test_} functions in this package}
     has_context = function() {
-      expect_is(ctx, "DBItest_context")
+      object <- ctx # object duplication fixes lazyeval error
```

Comment:
An omission in lazyeval, I guess. The expression `(ctx)` lives in the caller's frame, whereas `ctx` lives somewhere else. See https://github.com/hadley/testthat/pull/408 for more context. I'd be glad to see this resolved, but currently it's just a minor corner case that is easy to fix.